api/models.go

```diff
@@ -219,6 +228,15 @@ type SignMessageResponse struct {
 	Signature string `json:"signature"`
 }
 
+type WalletSendRequest struct {
```

Comment:
```suggestion
type SendPaymentRequest struct {
```
To be consistent with the response above